GVRf/Sample/gvr-video/src/org/gearvrf/video/VideoScript.java

```diff
@@ -632,6 +632,38 @@ public void run() {
             mainScene.getMainCameraRig().getOwnerObject()
                     .addChildObject(mPassThroughObject);
 
+            /*
+             * Selective objects to keep
```

Comment:
I think it would probably be better to set these extra reference counts at the point where mPlayPlauseButton / mFrontButton / &c are each initialized, not all in a 'add reference' block at the end.